lib/Sema/TypeCheckProtocol.cpp

```diff
@@ -379,7 +379,7 @@ matchWitnessDifferentiableAttr(DeclContext *dc, ValueDecl *req,
     bool foundExactConfig = false;
     Optional<AutoDiffConfig> supersetConfig = None;
     for (auto witnessConfig :
-         witnessAFD->getDerivativeFunctionConfigurations()) {
+           witnessAFD->getDerivativeFunctionConfigurations(/*NonLocal*/ false)) {
```

Comment:
Did you mean `NonPrimaryLookup`?
```suggestion
           witnessAFD->getDerivativeFunctionConfigurations(/*NonPrimaryLookup*/ false)) {
```